packages/beacon-state-transition/package.json

```diff
@@ -38,6 +38,7 @@
   "types": "lib/index.d.ts",
   "dependencies": {
     "@chainsafe/bls": "6.0.1",
+    "@chainsafe/lodestar-api": "^0.25.1",
```

Comment:
beacon-state-transition must not depend on the API. Define the `AttesterDuty` type again in that repo without importing it. You may skip the pubkey since it adds no info

Reply:
Ok, I'll copy in that AttesterDuty, but I think it might be more convenient and efficient to still have the pubkey there because otherwise we would have to re-map the pubkeys to the duties once we get the duties.

Reply:
It's def more efficient but since it's an operation you don't do 200_000 times it's doesn't really matter.